core/jvm/src/TimeZoneJvm.kt

```diff
@@ -54,6 +54,15 @@ public actual open class TimeZone internal constructor(internal val zoneId: Zone
     }
 }
 
+// Workaround for https://github.com/Kotlin/kotlinx-datetime/issues/149
+private val ZoneId.isFixedOffset: Boolean
+    get() = try {
+        // On older Android versions, this can throw even though it shouldn't
```

Comment:
Let's reference here a bug in the Android issue tracker so that we know when to remove the workaround.